### PR TITLE
Move wiki-documentation to Github

### PR DESCRIPTION
**Pull request by Yunir**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,93 @@
...
+After clicking the submit button, only checked combinations will run.
 
+![](https://wiki.jenkins.io/download/attachments/69271855/MatrixJobConfig.png?version=1&modificationDate=1375012523000&api=v2)
+
+Configuration:
+
+After installation a new type of parameter was added (Matrix Combinations Parameter).
+
+ 
+![](https://wiki.jenkins.io/download/attachments/69271855/MatrixPluginConfgJob.png?version=1&modificationDate=1375012524000&api=v2)
```
Comment (halkeye) [inline README.md]: You should use the .zip version of the exported doc so we don't have images still on the wiki

File: README.md [hunk 2]
```diff
@@ -1,11 +1,93 @@
-The plugin is inspired from the Matrix Reloaded Plugin and allows the user to choose which matrix combinations he
-wants to run.
+This plugin allows a user to choose which matrix combinations he wants to run, as opposed to the default behaviour where jenkins runs all combinations and translate it into a groovy condition such as:
```
Comment (timja) [inline README.md]: the first line of a readme should be a heading

File: README.md [hunk 3]
```diff
@@ -1,11 +1,93 @@
...
+After installation a new type of parameter was added (Matrix Combinations Parameter).
+
+ 
+![](https://wiki.jenkins.io/download/attachments/69271855/MatrixPluginConfgJob.png?version=1&modificationDate=1375012524000&api=v2)
+
+-   Configure your matrix job and add exactly one parameter with this type.
+-   Fill the name of the parameter.
+-   Define the Axes as usual
+
+# **Acknowledgements**
```
Comment (timja) [inline README.md]: # is a top level heading (title) and should only be on the first line of the file

File: README.md [hunk 4]
```diff
@@ -1,11 +1,93 @@
...
+
+Many thanks to Nathan Grunzweig for giving me the idea and help with the implementation of this plugin.  
+I would also like to thank Christian Wolfgang, who developed the Matrix Reloaded Plugin which gave me the inspiration to implement it.
+
+# Change Log
```
Comment (timja) [inline README.md]: # is a top level heading (title) and should only be on the first line of the file

I would suggest moving this section to a CHANGELOG.md file

**Comment by Yunir** [comment]
What labels should we attach to PR to update here (https://jenkins-wiki-exporter.jenkins.io/progress) that plugin has PR?

**Comment by halkeye** [comment]
> What labels should we attach to PR to update here (https://jenkins-wiki-exporter.jenkins.io/progress) that plugin has PR?

that isn't automatic so far. It either needs to be released, or updated manually. There's a PR in progress to support reading from a project, but I know I don't have access to that project to add it.

**Comment by halkeye** [comment]
I take it back, I can add it to the project!

File: CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
+#### Version 1.3.0 (3 Mar 2018)
```
Comment (timja) [inline CHANGELOG.md]: there should be a top-level heading 

```suggestion
# Changelog

#### Version 1.3.0 (3 Mar 2018)
```

File: README.md [hunk 6]
```diff
@@ -1,11 +1,29 @@
...
-wants to run.
+# Matrix combinations parameter plugin for Jenkins
 
-This plugin add a new type of parameter.
-When the user build a matrix project with this parameter type he gets a html table with checkbox for each combination.
-It gets from the user which combinations he wants to run, and translate it into a groovy condition such as:
+This plugin allows a user to choose which matrix combinations he wants to run, as opposed to the default behaviour where jenkins runs all combinations and translate it into a groovy condition such as:
 
-axis1=="axis1value1" && axis2=="axis2value1" || axis1=="axis1value2" && axis2=="axis2value2" || ...
+axis1=="axis1value1" && axis2=="axis2value1" || axis1=="axis1value2" && axis2=="axis2value2" || ...:
```
Comment (timja) [inline README.md]:
```suggestion
```groovy
axis1=="axis1value1" && axis2=="axis2value1" || axis1=="axis1value2" && axis2=="axis2value2" || ...:
```
```

**Comment by ikedam** [comment]
References:
* Instructions to move documents to github: https://jenkins.io/blog/2019/10/21/plugin-docs-on-github/#how-to-enable-github-documentation-for-your-plugin
* Old document:  https://wiki.jenkins-ci.org/display/JENKINS/Matrix+Combinations+Plugin (automatically redirects to the new document and so no longer available)
* New document: https://plugins.jenkins.io/matrix-combinations-parameter/ (No contents are available for now).

**Comment by ikedam** [comment]
I released matrix-combinations-plugin-1.3.1.
I believe the plugin documentation site will be updated soon.

**Pull request merged**